.github/workflows/docker-edge.yml

```
@@ -80,7 +80,24 @@
+        with:
+          context: .
+          load: true
+          file: packages/sync-server/docker/${{ matrix.os }}.Dockerfile
+          tags: actual-test
+          build-args: |
+            GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
+      - name: Test that the docker image boots
+        run: |
+          docker run --detach --publish=5006:5006 --network=host actual-test
+          sleep 5
+          curl -LI localhost:5006 -sS -w '%{http_code}\n' -fail --retry 10 --retry-delay 1 --retry-connrefused 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)